src/Hedgehog/Range.fs

```diff
@@ -54,20 +58,62 @@ module Range =
     //
 
     /// Construct a range which represents a constant single value.
+    ///
+    /// >>> Range.bounds x <| Range.singleton 5
+    /// (5, 5)
+    ///
+    /// >>> Range.origin <| Range.singleton 5
+    /// 5
```

Comment:
Well, it'll render as plain text. There's no supported suntax for this yet in .NET, because Doctest didn't exist before this PR.

There's [F# Formating](http://fsprojects.github.io/FSharp.Formatting/literate.html) that does something similar, but it has its own, verbose IMO,  syntax...